spec/vcr_setup.rb

```diff
@@ -0,0 +1,34 @@
+require 'vcr'
+require 'active_support/core_ext/string'
+
+def filter_body_field(c, field)
+  %w(request response).each do |part|
+    c.filter_sensitive_data("<#{field.underscore.upcase}>") do |interaction|
+      interaction[part].body.match(/"#{Regexp.quote(field)}":\s*"([^"]+)"/)&.captures&.first
+    end
+  end
+end
+
+def filter_header(c, placeholder, header)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)